appveyor.yml

```diff
@@ -4,17 +4,22 @@ configuration:
   - Debug
   - Release
 
+matrix:
+  allow_failures:
```

Comment:
Delete this? :P

Reply:
Looks like they are still failing unfortunately: https://ci.appveyor.com/project/robinlinden/n-e-s/builds/30045859

Reply:
Ah, I assumed they fixed something. Are we sure we're not doing anything that causes UB or something?

Might be better to disable the jobs entirely for now if we're not doing anything stupid. The failing ones run for >1h and the jobs run in sequence on every push, so if you rebase a PR twice, that's 2h before your CI runs again. :/

Reply:
There's also a Visual Studio 2019 Preview image we could try.

Reply:
Ok, I can try that. But can we merge this now to start with so we at least can get new PR merged?